recipes-bsp/tegra-binaries/tegra-helper-scripts/make-sdcard.sh

```diff
@@ -141,6 +141,27 @@ make_partitions() {
     return 0
 }
 
+create_filesystems() {
```

Comment:
The whitespace in these added lines should match up with what's in the rest of the file. Admittedly, the existing whitespace is somewhat peculiar (thanks, emacs!), with 4-space indentation but hard tabs on an 8-space boundary.